frontend/components/table/IRSCTable.tsx

```diff
@@ -0,0 +1,314 @@
+        JSON.stringify(updatedLocalStorage)
+      );
+    }
+  };
+
+  const columns = useMemo(
+    () => [
+      {
+        Header: () => format('form.header.requirement.label'),
+        accessor: 'requirement',
+      },
+      {
+        Header: format('form.header.comment.label'),
+        accessor: 'comment',
+        Cell: ({ row }: Cell<RequirementType>) => {
+          const [comment, setComment] = useState<string>(row.values.comment);
+          const [active, setActive] = useState(false);
+
+          const counter = (
+            <div
+              className={classNames('irsc-table-comment-counter', {
+                'counter-active': active,
+              })}
+            >
+              {comment.length}/100
+            </div>
+          );
+
+          return (
+            <div className="irsc-table-textarea">
+              <textarea
+                name="comment"
+                maxLength={100}
+                value={comment}
+                onChange={(event) => {
+                  setComment(event.target.value);
+                }}
+                onBlur={(event) => {
+                  handleUpdateField(
+                    row.original._id,
+                    'comment',
+                    event.target.value
+                  );
+                  setActive(false);
+                }}
+                onClick={() => setActive(true)}
+              />
+              {counter}
+            </div>
+          );
+        },
+      },
+      {
+        Header: format('form.header.fulfillment.label'),
+        accessor: 'fulfillment',
+        Cell: ({ row }) => {
+          return (
+            <div className="irsc-table-icons">
+              {row.values.fulfillment === 0 ? (
+                <FaCircleXmark
+                  fill="#CF0B0B"
+                  className="irsc-table-icon"
+                  onClick={() =>
+                    handleUpdateField(row.original._id, 'fulfillment', null)
+                  }
+                />
+              ) : (
+                <FaRegCircleXmark
+                  className="irsc-table-icon"
+                  onClick={() =>
+                    handleUpdateField(row.original._id, 'fulfillment', 0)
+                  }
+                />
+              )}
+              {row.values.fulfillment === 1 ? (
+                <FaCircleCheck
+                  fill="#048112"
+                  className="irsc-table-icon"
+                  onClick={() =>
+                    handleUpdateField(row.original._id, 'fulfillment', null)
+                  }
+                />
+              ) : (
+                <FaRegCircleCheck
+                  className="irsc-table-icon"
+                  onClick={() =>
+                    handleUpdateField(row.original._id, 'fulfillment', 1)
+                  }
+                />
+              )}
+            </div>
+          );
+        },
+      },
+    ],
+    [data.requirements.crossCutting]
+  );
+
+  // @ts-ignore
```

Comment:
I don't think this @ts-ignore and the one below affect anything. It should be possible to remove them